src/ProjectService.Data.Provider/IDataProvider.cs

```diff
@@ -12,5 +12,7 @@ public interface IDataProvider : IBaseDataProvider
         DbSet<DbProject> Projects { get; set; }
         DbSet<DbProjectFile> ProjectsFiles { get; set; }
         DbSet<DbProjectUser> ProjectsUsers { get; set; }
+        DbSet<DbTask> Task { get; set; }
```

Comment:
use names with 's' on the end, Tasks and TaskProperties

Reply:
done